src/index.html

```diff
@@ -4,9 +4,442 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    > 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header body__flexbox">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="list nav__flexbox">
+          <li class="list__first">
+            <a href="#Apple" class="list__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#Samsung" class="list__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#Smartphones" class="list__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a
+              href="#L&C"
+              class="list__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#Gadgets" class="list__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#Tablets" class="list__link">
+              Tablets
+            </a>
+          </li>
+          
+          <li class="list__item">
+            <a href="#Photo" class="list__link">
+              Photo
+            </a>
+          </li>
+          <li class="list__item">
+            <a href="#Video" class="list__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="block main__row">
+        <div class="card block__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="apple imac"
+          >
+          <a class="card__name" href="#">
```

Comment:
```suggestion
          >
          
          <a class="card__name" href="#">
```
pay attention to the code style, fix it